calculate_average_ebarlas.sh

```diff
@@ -15,6 +15,7 @@
 #  limitations under the License.
 #
 
+source "$HOME/.sdkman/bin/sdkman-init.sh"
 sdk use java 21.0.1-graalce
```

Comment:
```suggestion
sdk use java 21.0.1-graalce 1>&2
```

This will make sure that the additional output doesn't interfere with the test assertion.